apps/validation-messages-example-app/src/app/app.component.html

```diff
@@ -1,28 +1,51 @@
-<form [formGroup]="form">
-  <div class="dialog__content">
-    <mat-form-field appearance="fill">
+<form [formGroup]="form" class="form">
+  <div>
+    <mat-form-field>
+      <mat-label>Email</mat-label>
+      <input matInput type="email" formControlName="email" />
+      <mat-error>
+        <!-- component retrieves form control from mat-form-field -->
+        <!-- global messages are extended/overridden by a local config -->
+        <ng-validation-messages
+          [validationMessages]="{
```

Comment:
I think input should be called errorsMessages

Reply:
I've changed its name to errorMessages